toolium/visual_test.py

```diff
@@ -161,7 +161,7 @@ def assert_screenshot(self, element, filename, file_suffix=None, threshold=0, ex
         DriverWrappersPool.visual_number += 1
 
         # Determine whether we should save the baseline image
-        if self.save_baseline or not os.path.exists(baseline_file):
+        if self.save_baseline and not os.path.exists(baseline_file):
```

Comment:
This is the intentional behavior, baseline should be saved in two cases:
* when 'save' mode is enabled
* when 'save' mode is disabled, but there are no previous baseline

In 'save' mode, screenshots are not compared. It is just used to save baselines.